ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2018.3.9f1
+m_EditorVersion: 2018.3.8f1
```

Comment:
```suggestion
m_EditorVersion: 2018.3.9f1
```

Reply:
I think it might be best to leave this as-is (or at least not turn back the editor version clock)

Reply:
Yeah sorry, that snug in :/